pkgs/applications/misc/resumed/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, buildNpmPackage, fetchFromGitHub }:
+
+buildNpmPackage rec {
```

Comment:
Apparently `buildNpmPackage` does not support the `(finalAttrs: { ... })` format?

Reply:
We don't need this here